alert-common/src/main/java/com/synopsys/integration/alert/common/workflow/task/StartupScheduledTask.java

```diff
@@ -31,18 +31,26 @@
 
     private final TaskManager taskManager;
 
-    public StartupScheduledTask(final TaskScheduler taskScheduler, final String taskName, final TaskManager taskManager) {
+    public StartupScheduledTask(TaskScheduler taskScheduler, String taskName, TaskManager taskManager) {
         super(taskScheduler, taskName);
         this.taskManager = taskManager;
     }
 
     public abstract String scheduleCronExpression();
 
+    public boolean shouldScheduleTask() {
```

Comment:
I think adding enable/disable functionality would be better here. If we plan on adding Tasks in theUI, we'll need enable/disable anyway.